lib/experimental/interactivity-api/script-loader.php

```diff
@@ -11,22 +11,28 @@
  * @param WP_Scripts $scripts WP_Scripts instance.
  */
 function gutenberg_register_interactivity_scripts( $scripts ) {
-	gutenberg_override_script(
-		$scripts,
-		'wp-interactivity-runtime',
-		gutenberg_url(
-			'build/block-library/interactive-blocks/interactivity.min.js'
-		),
-		array( 'wp-interactivity-vendors' )
-	);
+	// When in production, use the plugin's version as the default asset version;
+	// else (for development or test) default to use the current time.
+	$default_version = defined( 'GUTENBERG_VERSION' ) && ! SCRIPT_DEBUG ? GUTENBERG_VERSION : time();
 
-	gutenberg_override_script(
-		$scripts,
-		'wp-interactivity-vendors',
-		gutenberg_url(
-			'build/block-library/interactive-blocks/vendors.min.js'
-		)
-	);
+	foreach ( array( 'vendors', 'runtime' ) as $script_name ) {
```

Comment:
I wanted to explore how we could automate registering the runtime based on the asset files that we use for all other scripts built with webpack. In particular, the version generate is very important as it only changes when the file's content changes, so it's essential for proper caching in WordPress core.